packages/flame/src/Alert/Alert.tsx

```diff
@@ -84,14 +53,26 @@ export const Alert: React.FC<AlertProps & SpaceProps> = ({
   if (isHidden) return null;
 
   return (
-    <AlertWrapper type={type} {...restProps}>
-      <Flex flex="1" alignItems="flex-start">
-        {icon && (
-          <Flex alignItems="center" pr={2} pt="2px">
-            {icon}
+    <Box
+      css={css({
+        display: 'flex',
+        justifyContent: 'space-between',
+        alignItems: 'flex-start',
+        boxShadow:
+          '0 0 0 1px rgba(0, 0, 0, 0.06), 0 3px 6px 0 rgba(0, 0, 0, 0.03), 0 1px 2px 0 rgba(0, 0, 0, 0.1)',
+        borderTop: '3px solid',
+        borderRadius: 'radius-2',
+        px: 3,
+        py: 2,
+        variant: `alertVariants.${type}`,
+      })}
+      {...restProps}
+    >
+      <Flex flex="1">
+        <Box flex="1" css={css({ position: 'relative', pl: 5 })}>
+          <Flex className="fl-alert__icon" css={{ position: 'absolute', left: '0px', top: '2px' }}>
```

Comment:
Should this be using `css={css()}` instead even tho we're not using theme values?

Reply:
doesn't matter